ui/src/components/Link/index.tsx

```diff
@@ -1,36 +1,38 @@
-/* eslint-disable jsx-a11y/anchor-is-valid */ // Disabled due to NextLink using an anchor with an onClick
-/* eslint-disable jsx-a11y/click-events-have-key-events */ // Disabled due to having an onClick & not a key event listener
-import Link from 'next/link';
 import React from 'react';
+import Link, { LinkProps } from 'next/link';
 
-type Props = {
-    href: string;
-    title?: string;
-    className?: string;
-    scroll?: boolean;
+// custom props will override default attributes if both are provided
+type CustomProps = {
     openNew?: boolean;
     ariaLabel?: string;
```

Comment:
Well, I just added those optional custom props so I don't need to edit all the occurrences in all the files where this component was used. I would have removed those custom props otherwise.

Since "ariaLabel" prop refers to "aria-label" attribute (which is not a required attribute), then "ariaLabel" prop shouldn't be required. Same goes for "openNew" prop, which refers to the "target" attribute, which is also not required.